test/tpmclient/SetLocality.c

```diff
@@ -27,6 +27,7 @@
 
 #include "sapi/tpm20.h"
 #include "sample.h"
+#include "../integration/sapi-util.h"
```

Comment:
Why not set -I via the makefile?

Reply:
I had a version that did that. I figured a relative path here wasn't much worse. The end goal is to get rid of this file regardless. Really it should be built into the utility library in the integration directory. Just haven't got around to it yet.